src/components/manifold-plan-table/manifold-plan-table.tsx

```diff
@@ -420,48 +403,58 @@ export class ManifoldPlanTable {
                 />
               </p>
             </div>,
-            Object.values(this.planFeatures[plan.id]).map((feature) => {
-              // fixed feature
-              if (feature && this.productFeatures.fixed[feature.label]) {
-                return <FixedFeature displayValue={(feature as PlanFixedFeature).displayValue} />;
-              }
-
-              // metered feature
-              if (feature && this.productFeatures.metered[feature.label]) {
-                return <MeteredFeature feature={feature as PlanMeteredFeature} />;
-              }
-
-              // configurable
-              if (feature && this.productFeatures.configurable[feature.label]) {
-                const configurableFeature = feature as PlanConfigurableFeature;
-                return this.displayConfigurable({
-                  planID: plan.id,
-                  feature: configurableFeature,
-                });
-              }
-
-              // undefined / disabled feature
-              return (
-                <div class="ManifoldPlanTable__Cell ManifoldPlanTable__Cell--Body">
-                  <span class="ManifoldPlanTable__Cell__Disabled">•</span>
-                </div>
-              );
-            }),
-            <div
-              class="ManifoldPlanTable__Cell ManifoldPlanTable__Cell--Body"
-              data-row-last
-              data-column-last={lastColumn}
+            <form
+              action={this.baseUrl}
+              onSubmit={this.onSubmit(plan.id)}
+              style={{ display: 'contents' }}
```

Comment:
Could you move this style into maybe `ManifoldPlanTable__PlanColumn` or something in `_plan.scss`?

I don’t think a user would realistically want to override this, but it still breaks our pattern of CSS class-level specificity (we try and not inline properties as a general pattern to keep things flexible)

Reply:
Absolutely